Code/Tools/ProjectManager/Source/ProjectButtonWidget.cpp

```diff
@@ -197,7 +198,8 @@ namespace O3DE::ProjectManager
 
             QMenu* menu = new QMenu(this);
             menu->addAction(tr("Edit Project Settings..."), this, [this]() { emit EditProject(m_projectInfo.m_path); });
-            menu->addAction(tr("Build"), this, [this]() { emit BuildProject(m_projectInfo); });
+            menu->addAction(tr("Auto-build"), this, [this]() { emit BuildProject(m_projectInfo); });
```

Comment:
what does the "auto" mean here? 
maybe "default"?

Reply:
"Auto-build" was the recommendation from UX, to mean the Project Manager "automatically build for you"

Reply:
I really find that confusing, its not automatic right? you have to trigger the build
It also wont detect changes and trigger based on it, the user always has to trigger it.
I am confused on what the "automatic" part is here.

Reply:
The automatic part is that it runs configure and build steps for you and fills in all the config settings with defaults.

Reply:
what about "Build default settings"?

Reply:
"Build default settings" can mean that there are additional settings users need to configure before building. how about "Build Now"?

Reply:
I prefer "Build" than "Build Now"
"Build Now" makes me wonder what other option is going to be there...

Reply:
I'll use "Build Now" for now (haha),  using "Build" alone would also be good except it looks redundant when the button is called "Build" too.  However, we can make the Project Menu just "Build

Reply:
I'll compromise and use "Build" in the Project menu and "Build Now" in the "Build Project" menu.